bg_atlasgen/validate_atlases.py

```diff
@@ -152,16 +153,20 @@ def validate_atlas(atlas_name, version, validation_functions):
     if not updated:
         update_atlas(atlas_name)
 
-    # list to store the errors of the failed validations
+    # dictionary of lists to store the errors of the failed validations
     failed_validations = {atlas_name: []}
     successful_validations = {atlas_name: []}
 
     for i, validation_function in enumerate(validation_functions):
         try:
             validation_function(BrainGlobeAtlas(atlas_name))
-            successful_validations[atlas_name].append(validation_function)
+            successful_validations[atlas_name].append(
+                validation_function.__name__
+            )
```

Comment:
```suggestion
            validations[atlas_name].append(
                validation_function.__name__, None
            )
```

(:arrow_up: Just a sketch... and just a discussion point)

Should we combine the two lists while we're refactoring this, and have them have the same format (`str(error)` if failed, and `None` if valid)? The advantage would be simplicity
* just one file
* shorter and easier to read code

But it's entirely possible there's value in keeping the things separate? What do you think?

Reply:
Thanks for this suggestion! I was also thinking about doing this but wanted to have a chat first, as I wasn't sure how to implement it. I think it would be much better to have only one file with all the necessary information.

Reply:
@alessandrofelder, I implemented these changes and tested them locally and on the HPC. Could you check them, please? 🙂

Reply:
Thanks @viktorpm - As per [our developer's guide around PRs](https://brainglobe.info/community/developers/index.html#pull-requests), you have the liberty to merge if your reviewer approves with optional comments without asking for another round of review :) I have had another look though and looks great!